spyder/plugins/editor/widgets/tests/test_introspection.py

```diff
@@ -25,6 +27,35 @@
 LOCATION = osp.realpath(osp.join(os.getcwd(), osp.dirname(__file__)))
 
 
+@pytest.mark.slow
+@pytest.mark.use_introspection
+@flaky(max_runs=3)
+@pytest.mark.skipif(os.name == 'nt' or not PY2,
+                    reason="Times out on AppVeyor and fails on PY3/PyQt 5.6")
+def test_calltip(lsp_codeeditor, qtbot):
+    """Test that the calltip in editor is hidden when matching ')' is found."""
+    code_editor, _ = lsp_codeeditor
+
+    text = 'a = [1,2,3]\n(max'
+    # Set text to start
+    code_editor.set_text(text)
```

Comment:
Please add

```python
with qtbot.waitSignal(code_editor.lsp_response_signal, timeout=30000):
        code_editor.document_did_change()
```

below this line.

Reply:
Why is this necessary? Setting the text now calls document dis change in the code editor

Reply:
Right, sorry, you can ignore this comment @steff456.